src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -384,7 +434,7 @@ public void gatherXp()
     {
         for (@NotNull EntityXPOrb orb : getXPOrbsOnGrid())
         {
-            addExperience(orb.getXpValue());
+            addExperience(orb.getXpValue() / 2);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Cast one of the operands of this division operation to a "double". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2184)